### ISSUE TITLE
Chrome Extension download link is broken

### ISSUE THREAD

**Issue by ragultheepanr** [iqnivek/save_tabbed_images#48]
The download link provided for the extension https://chrome.google.com/webstore/detail/save-tabbed-images/hhcoikfhkkadkgklepjkfgafmjoggefh is broken. It leads me to a page which displays a message as stated below.

404. That’s an error.

The requested URL was not found on this server. That’s all we know.

![Capture](https://user-images.githubusercontent.com/22179225/83184002-c96d3d80-a0f6-11ea-94d6-2a0954d6c0f4.PNG)

**Comment by kevinsqi** [comment]
Yeah; google sent out an email to a lot of extension authors asking to reduce permissions or the extension would be yanked from the store. This seems reasonable, but their email doesn't really contain any info on how you should reduce permissions.. I haven't gotten a chance to look at it yet, so it's gone right now. 


```
Your item did not comply with the following section of our Program Policies:


"User Data Privacy"

Your product violates the "Use of Permissions" section of the policy, which requires that you:

Request access to the narrowest permissions necessary to implement your product’s features or services.
If more than one permission could be used to implement a feature, you must request those with the least access to data or functionality.
Don't attempt to "future proof" your product by requesting a permission that might benefit services or features that have not yet been implemented.

Please make the necessary changes within 14 days in order to avoid removal.
```

**Comment by ragultheepanr** [comment]
I was still able to download the file from your github and install the extension as unpacked extension using developer mode. Will Google still be able detect and delete the extension  from my chrome, since I am still using it without reducing the permissions?

**Comment by ragultheepanr** [comment]
I was still able to download the file from your github and install it as an unpacked extension using developer mode. But, sometime I get an error and the extension gets turned of automatically, is there anything I can do?

**Comment by ghost** [comment]
@ragultheepanr how did you do that I just downloaded it as zip unpacked it and tried to install in chrome but it gives me error. Please share how to install it in chrome.
![Untitled](https://user-images.githubusercontent.com/66911679/84596047-f94b6d80-ae78-11ea-8759-b89355ca169d.png)

**Comment by KahPal** [comment]
Any chance for instructions how to add working extension manually?

**Comment by mruac** [comment]
You can download a copy of .crx file from [CRX4Chrome](https://www.crx4chrome.com/extensions/hhcoikfhkkadkgklepjkfgafmjoggefh/) and install it as usual.
